proposals/simd/webassembly-opcodes.md

```diff
@@ -0,0 +1,275 @@
+| `f64x2.replaceLane(a: v128, i: LaneIdx2, x: f64) -> v128` | [f64x2.replaceLane](portable-simd.md#replace-lane-value) |
+| `f64x2.add(a: v128, b: v128) -> v128` | [f64x2.add](portable-simd.md#addition) |
+| `f64x2.sub(a: v128, b: v128) -> v128` | [f64x2.sub](portable-simd.md#subtraction) |
+| `f64x2.mul(a: v128, b: v128) -> v128` | [f64x2.mul](portable-simd.md#multiplication) |
+| `f64x2.neg(a: v128) -> v128` | [f64x2.neg](portable-simd.md#negation) |
+| `f64x2.eq(a: v128, b: v128) -> b64x2` | [f64x2.equal](portable-simd.md#equality) |
+| `f64x2.ne(a: v128, b: v128) -> b64x2` | [f64x2.notEqual](portable-simd.md#non-equality) |
+| `f64x2.lt(a: v128, b: v128) -> b64x2` | [f64x2.lessThan](portable-simd.md#less-than) |
+| `f64x2.le(a: v128, b: v128) -> b64x2` | [f64x2.lessThanOrEqual](portable-simd.md#less-than-or-equal) |
+| `f64x2.gt(a: v128, b: v128) -> b64x2` | [f64x2.greaterThan](portable-simd.md#greater-than) |
+| `f64x2.ge(a: v128, b: v128) -> b64x2` | [f64x2.greaterThanOrEqual](portable-simd.md#greater-than-or-equal) |
+| `f64x2.abs(a: v128) -> v128` | [f64x2.abs](portable-simd.md#absolute-value) |
+| `f64x2.min(a: v128, b: v128) -> v128` | [f64x2.min](portable-simd.md#nan-propagating-minimum) |
+| `f64x2.max(a: v128, b: v128) -> v128` | [f64x2.max](portable-simd.md#nan-propagating-maximum) |
+| `f64x2.div(a: v128, b: v128) -> v128` | [f64x2.div](portable-simd.md#division) |
+| `f64x2.sqrt(a: v128) -> v128` | [f64x2.sqrt](portable-simd.md#square-root) |
+| `f64x2.convert_s/i64x2(a: v128) -> v128` | [f64x2.fromSignedInt](portable-simd.md#integer-to-floating-point) |
+| `f64x2.convert_u/i64x2(a: v128) -> v128` | [f64x2.fromUnsignedInt](portable-simd.md#integer-to-floating-point) |
+
+## `b8x16` operations
+| WebAssembly | Portable SIMD |
+|:------------|:--------------|
+| `b8x16.build(x: i32[16]) -> b8x16` | [b8x16.build](portable-simd.md#build-vector-from-individual-lanes) |
+| `b8x16.splat(x: i32) -> b8x16` | [b8x16.splat](portable-simd.md#create-vector-with-identical-lanes) |
+| `b8x16.extractLane(a: b8x16, i: LaneIdx16) -> i32` | [b8x16.extractLane](portable-simd.md#extract-lane-as-a-scalar) |
+| `b8x16.replaceLane(a: b8x16, i: LaneIdx16, x: i32) -> b8x16` | [b8x16.replaceLane](portable-simd.md#replace-lane-value) |
+| `b8x16.and(a: b8x16, b: b8x16) -> b8x16` | [b8x16.and](portable-simd.md#logical-and) |
+| `b8x16.or(a: b8x16, b: b8x16) -> b8x16` | [b8x16.or](portable-simd.md#logical-or) |
+| `b8x16.xor(a: b8x16, b: b8x16) -> b8x16` | [b8x16.xor](portable-simd.md#logical-xor) |
+| `b8x16.not(a: b8x16) -> b8x16` | [b8x16.not](portable-simd.md#logical-not) |
+| `b8x16.anyTrue(a: b8x16) -> i32` | [b8x16.anyTrue](portable-simd.md#any-lane-true) |
+| `b8x16.allTrue(a: b8x16) -> i32` | [b8x16.allTrue](portable-simd.md#all-lanes-true) |
+
+## `b16x8` operations
+| WebAssembly | Portable SIMD |
+|:------------|:--------------|
+| `b16x8.build(x: i32[8]) -> b16x8` | [b16x8.build](portable-simd.md#build-vector-from-individual-lanes) |
```

Comment:
Does bools have uses beyond holding/manipulating results of logical and relational operations?
Wondering if there is a need to have support for building bools, since we intend to have bool consts.